### DISCUSSION TITLE
Error 500 after upgrade

### DISCUSSION THREAD

**Discussion by LazyNCoder** [YOURLS/YOURLS#3200] [Q&A]
Hello guys,

I've been using yourls v1.7.2 for years now but decided to upgrade to v1.8.2 for better security and performance. 

My admin panel works but the redirection is not. It throws error 500:
[18-Jan-2022 07:17:41 UTC] PHP Fatal error:  U in short/includes/vendor/aura/sql/src/AbstractExtendedPdo.php on line 558

I don't know what to do. My v1.7.2 still works without problem. Tried to change the db to utf8_general_ci but it didn't help. 

Any suggestions?

**Comment by ozh** [comment]
"PHP Fatal error: U" ??

**Reply by LazyNCoder** [reply]
Believe me, it's what it says in my error_log file  😂

**Reply by ozh** [reply]
Well, that's a new one. Most likely : incorrectly uploaded file or files. Upload new files again and overwrite everything ?

**Reply by LazyNCoder** [reply]
I tried that but it didn't work. Just copied my config and .htaccess from the old installation. Still I'm getting the same error.

**Comment by LazyNCoder** [comment]
I managed to get the full error message as there was a mistake in my php configuration. Now it's:
[25-Jan-2022 06:36:18 UTC] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'mydb.yourls_log' doesn't exist in short/includes/vendor/aura/sql/src/AbstractExtendedPdo.php:558

**Reply by ozh** [reply]
What are the ACTUAL db names and db prefix ?

**Reply by ehguille** [reply]
Hi there,

Just in case it helps.

I'm using 1.8.2 and also get a TOO_MANY_REDIRECTS error.

I've been checking my logs and get the following message:

```
[Fri Feb 04 15:48:56.564399 2022] [php7:error] [pid 1582] [client ***.**.**.123:51068] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yourls_db.yourls_options' doesn't exist in /var/www/sites/mydomain/web/html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php:558\nStack trace:\n#0 /var/www/sites/mydomain/web/html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php(558): PDOStatement->execute()\n#1 /var/www/sites/mydomain/web/html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php(412): Aura\\Sql\\AbstractExtendedPdo->perform()\n#2 /var/www/sites/mydomain/web/html/includes/Database/Options.php(110): Aura\\Sql\\AbstractExtendedPdo->fetchOne()\n#3 /var/www/sites/mydomain/web/html/includes/functions-options.php(25): YOURLS\\Database\\Options->get()\n#4 /var/www/sites/mydomain/web/html/includes/functions.php(501): yourls_get_option()\n#5 /var/www/sites/mydomain/web/html/includes/functions.php(483): yourls_get_current_version_from_sql()\n#6 /var/www/sites/mydomain/web/html/includes/Config/Init.php(95): yourls_upgrade_is_needed()\n#7 /var/www/sites/mydomain/web/html/includes/ in /var/www/sites/mydomain/web/html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php on line 558
```
Obviously, yourls_option exists:

![image](https://user-images.githubusercontent.com/55703981/152601711-18d3ddf5-d8d1-4197-a340-cfbe853084d3.png)

The full ~~history~~ story here: https://github.com/YOURLS/YOURLS/issues/3210
